docs/csharp/language-reference/operators/bitwise-and-shift-operators.md

```diff
@@ -7,6 +7,7 @@ f1_keywords:
   - "~_CSharpKeyword"
   - "<<_CSharpKeyword"
   - ">>_CSharpKeyword"
+  - ">>>_CSharpKeyword"
```

Comment:
This should fix the `F1` navigation from Visual Studio. The compound assignment is already in the list.